python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java

```diff
@@ -218,29 +225,32 @@ private void launchIPythonKernel(int ipythonPort)
     watchDog = new ExecuteWatchdog(ExecuteWatchdog.INFINITE_TIMEOUT);
     executor.setWatchdog(watchDog);
 
-    String py4jLibPath = null;
-    if (System.getenv("ZEPPELIN_HOME") != null) {
-      py4jLibPath = System.getenv("ZEPPELIN_HOME") + File.separator
-          + PythonInterpreter.ZEPPELIN_PY4JPATH;
-    } else {
-      Path workingPath = Paths.get("..").toAbsolutePath();
-      py4jLibPath = workingPath + File.separator + PythonInterpreter.ZEPPELIN_PY4JPATH;
-    }
-    if (additionalPythonPath != null) {
-      // put the py4j at the end, because additionalPythonPath may already contain py4j.
-      // e.g. PySparkInterpreter
-      additionalPythonPath = additionalPythonPath + ":" + py4jLibPath;
-    } else {
-      additionalPythonPath = py4jLibPath;
+    if (addBuiltinPy4j) {
```

Comment:
nit `addBuiltinPy4j` is a bit confusing? `useBuiltinPy4j`? `addBuiltinPy4jPath`?